ContentProtection/OfflinePlayReadyAndWidevine/src/main/java/sample/OfflinePlayReadyAndWidevine.java

```diff
@@ -515,17 +434,16 @@ private static ContentKeyPolicyPlayReadyConfiguration configurePlayReadyLicenseT
      * @param streamingEndpoint The streaming endpoint.
      * @return DASH url.
      */
-    private static String getDASHStreamingUrl(MediaManager manager, String resourceGroup, String accountName, String locatorName, StreamingEndpoint streamingEndpoint) {
+    private static String getDASHStreamingUrl(MediaServicesManager manager, String resourceGroup, String accountName, String locatorName, StreamingEndpoint streamingEndpoint) {
         String dashPath = "";
 
         ListPathsResponse paths = manager.streamingLocators()
-                .listPathsAsync(resourceGroup, accountName, locatorName)
-                .toBlocking().first();
+                .listPaths(resourceGroup, accountName, locatorName);
 
         for (StreamingPath path : paths.streamingPaths()) {
```

Comment:
I actually don't prefer to use the listpaths method myself, I like to build the URI up to point to the HLS and DASH CMAF instances only.  
Take a look at this method I use in the Node.js library that builds the URI more deterministically 
https://github.com/Azure-Samples/media-services-v3-node-tutorials/blob/f7481d5691dd55359ab3668d03d873c32cc0c251/Common/Encoding/encodingJobHelpers.ts#L432